fabric-transfer-api-v1/build.gradle

```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-transfer-api-v1"
-version = getSubprojectVersion(project, "1.0.0")
+version = getSubprojectVersion(project, "1.1.0-pre.05")
```

Comment:
Can you leave the versions alone please? This is not right.

Reply:
No worries, this was just for pre-merge testing in case multiple mods make use of it.